lib/apiRequest.js

```diff
@@ -0,0 +1,42 @@
+const fetch = require("node-fetch");
+const getTime = require('./time');
+const generateSignature = require('./generateSignature');
+const CONSTANTS = require("../utils/constants");
+  
+async function apiRequest(apiSecret, apiKey, method, path, body){  
```

Comment:
I would read the `apiKey` and `apiSecret` from env variables instead of getting them here as function parameters. They do not change per API call, so why getting them from each call